### PR TITLE
[cudaaligner] Hirschberg + Myers algorithm

### PR DESCRIPTION
**Pull request by ahehn-nv**

### CODE REVIEW

File: cudaaligner/src/batched_device_matrices.cuh [hunk 1]
```diff
@@ -23,9 +23,9 @@ namespace claragenomics
 namespace cudaaligner
 {
 
-__device__ inline bool error(int t)
+__device__ inline bool error(int x, int y)
```
Comment (tijyojwad) [inline cudaaligner/src/batched_device_matrices.cuh]: to be consistent, let's change the `int` to `int32_t`?

**Comment by tijyojwad** [comment]
@ahehn-nv can you also change the class instantiated in`aligner.cpp` to be HirschbergMyers?

File: cudaaligner/src/batched_device_matrices.cuh (hunk 1 above)
Comment (ahehn-nv) [reply cudaaligner/src/batched_device_matrices.cuh]: good point. fixed.

**Comment by ahehn-nv** [comment]
Yes, changed the default aligner for global alignment to HirschbergMyers.

**Pull request merged**